defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-postgresql_support_libpq_version: "9.5.*.pgdg14.04+2"
+postgresql_support_libpq_version: "9.5.*.pgdg*"
```

Comment:
My assumption is that PostgreSQL is installing this dependency itself. If I use this version string to provision the DRIVER database virtual machine, the following task doesn't get marked as changed:

```
TASK [azavea.postgresql-support : Install client API libraries for PostgreSQL] *
ok: [database]
```